guides/common/modules/proc_migrating-to-a-new-server.adoc

```diff
@@ -14,8 +14,8 @@ If you recently created an offline backup, you can perform an incremental backup
 
 .Procedure
 * Perform an offline or incremental backup on the source machine:
-** To perform an offline backup, see xref:Backing-Up-Server-and-Proxy_{context}[].
-** To perform an incremental backup, see xref:Performing-an-Incremental-Backup_{context}[].
+** To perform an offline backup, see xref:"Backing-Up-Server-and-Proxy"_{context}[].
```

Comment:
Why are you adding double quotes here? I don't think we have an example where we do/need this.